### DISCUSSION TITLE
What exactly are profile templates and how do I set one up?

### DISCUSSION THREAD

**Discussion by isle9** [filips123/PWAsForFirefox#319] [Q&A] [answered]
I wasn't able to find any info in the repository.

**Comment by filips123** [comment, marked as answer]
In the extension settings, you can enter a custom path in the "Default profile template" option. You can also specify a template directory directly when creating a new profile. When you then create a new profile, all files in that template directory will be copied to your new profile. This can be used for example to set up profile settings once and then just re-use them as template for other profiles.
